pipelines: Store details about specs broken on develop

scottwittenburg opened the pull request:
Instead of storing an empty file as the entry in the broken specs list, store the full spec yaml as well as links to the pipeline and job that caused the spec to be added to the list.

scottwittenburg committed:
pipelines: Store details about specs broken on develop

becker33 commented:
@scottwittenburg there's a conflict preventing this from running tests/being merged

scottwittenburg commented:
> @scottwittenburg there's a conflict preventing this from running tests/being merged

Ok, I'll rebase it, thanks.  I somehow missed that when I looked at this yesterday.

scottwittenburg commented:
@becker33 I saw you merged `develop` here, thanks, but it didn't fix the style issues, so I rebased and fixed them manually.  If you can re-approve when you get a chance, I'll merge.  Thanks!
